corehq/apps/data_interfaces/models.py

```diff
@@ -214,7 +214,7 @@ def by_domain(cls, domain, workflow, active_only=True):
             workflow=workflow,
             deleted=False,
             **additional_filters
-        )
+        ).order_by('name', 'id')
```

Comment:
could we apply this to only the view and not the model? in cases where order doesn't matter this adds a slight unnecessary performance hit (like when we lookup rules to run them post case save) which i'd like to avoid

Reply:
sure => c30c91f8cce85ab171549c42a430da5dd768c308